### PR TITLE
Batch component update

### PR DESCRIPTION
**Pull request by alexkilgour**
Batch update of components to make sure they are all using the latest `brand-context`. Includes the removal of old extended components to be replaced with new branded global components.

### CODE REVIEW

File: toolkits/global/packages/global-ad/README.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ Catering for ad slots, specifically:
 
 Minimum height set for leaderboard and mpu slots to prevent reflow of layout.
 
+## Branding
+
+The `global-ad` component currently uses the `DEFAULT` brand only.
+
+```scss
+// Inlcude this with your settings
```
Comment (ricky-lau) [inline toolkits/global/packages/global-ad/README.md:15]: typo of include

File: toolkits/global/packages/global-ad/README.md [hunk 2]
```diff
@@ -7,6 +7,18 @@ Catering for ad slots, specifically:
...
+@import '@springernature/global-ad/scss/10-settings/default';
+
+// Incude this with your other components
```
Comment (ricky-lau) [inline toolkits/global/packages/global-ad/README.md:18]: typo of include

File: toolkits/global/packages/global-autocomplete/README.md [hunk 3]
```diff
@@ -2,6 +2,18 @@
 
 Basic autocomplete function that listens to key events on a text input and fetches suggestions from a specified endpoint.
 
+## Branding
+
+The `global-autocomplete` component currently uses the `DEFAULT` brand only.
+
+```scss
+// Inlcude this with your settings
```
Comment (ricky-lau) [inline toolkits/global/packages/global-autocomplete/README.md:10]: typo

File: toolkits/global/packages/global-autocomplete/README.md [hunk 4]
```diff
@@ -2,6 +2,18 @@
...
+@import '@springernature/global-autocomplete/scss/10-settings/default';
+
+// Incude this with your other components
```
Comment (ricky-lau) [inline toolkits/global/packages/global-autocomplete/README.md:13]: typo

File: toolkits/global/packages/global-banner/README.md [hunk 5]
```diff
@@ -2,6 +2,20 @@
...
+
+To include `global-banner` in your application, you need to choose **ONE** brand from those available. The `DEFAULT` brand is included in all other brands, and any settings that are not configured will fall back to default.
+
+```scss
+// Pick ONE of the brands below to include
+@import '@springernature/global-banner/scss/10-settings/default';
+@import '@springernature/global-banner/scss/10-settings/springer';
+@import '@springernature/global-banner/scss/10-settings/nature';
+
+// Incude this with your other components
```
Comment (ricky-lau) [inline toolkits/global/packages/global-banner/README.md:15]: typo

File: toolkits/global/packages/global-forms/README.md [hunk 6]
```diff
@@ -1,3 +1,15 @@
 # Global Forms
 
 This module gives you some generic styles for text, checkbox, radio button, textarea and select inputs.
+
+## Branding
+
+The `global-forms` component currently uses the `DEFAULT` brand only.
+
+```scss
+// Inlcude this with your settings
```
Comment (ricky-lau) [inline toolkits/global/packages/global-forms/README.md:10]: typo

File: toolkits/global/packages/global-forms/README.md [hunk 7]
```diff
@@ -1,3 +1,15 @@
...
+@import '@springernature/global-forms/scss/10-settings/default';
+
+// Incude this with your other components
```
Comment (ricky-lau) [inline toolkits/global/packages/global-forms/README.md:13]: typo

File: toolkits/global/packages/global-menu/README.md [hunk 8]
```diff
@@ -2,6 +2,18 @@
 
 Create common navigation patterns for list elements.
 
+## Branding
+
+The `global-menu` component currently uses the `DEFAULT` brand only.
+
+```scss
+// Inlcude this with your settings
```
Comment (ricky-lau) [inline toolkits/global/packages/global-menu/README.md:10]: typo

File: toolkits/global/packages/global-menu/README.md [hunk 9]
```diff
@@ -2,6 +2,18 @@
...
+@import '@springernature/global-menu/scss/10-settings/default';
+
+// Incude this with your other components
```
Comment (ricky-lau) [inline toolkits/global/packages/global-menu/README.md:13]: typo

Comment (alexkilgour) [reply toolkits/global/packages/global-menu/README.md:13]: 🤦

**Pull request merged**